### ISSUE TITLE
backend NVIDIA disabled - Miner crash on Vega 56

### ISSUE THREAD

**Issue by leokillbill** [fireice-uk/xmr-stak#1253]
Hi Guys,
I am new to Mining and this is the first time setting up a system following some online guide. 
Anyway, I am using a Vega 56 card but the Miner is crashing on me with a NVIDIA error.
All I did is to install the AMD Blockchain Driver and double clicked on the xmr-stak.exe. 
I have tried reinstalled the Blockchain Driver, disable and re-enable the graphic card after restart, still getting the same result:
Start Mining: Monero
WARNING: NVIDIA Insufficient Driver!
WARNING: backend NVIDIA disabled
And then the Miner will just crash before it even starts to mine.

Please advise. Any suggestions welcome.
Thank you all very much!

Please provide as much as possible information to reproduce the issue.

# Basic information
  - Pentium.
  - Sapphire Vega 56, just installed the first one.

# Compile issues
  - Windows 10 CB 1709
  ```
  add **all** commands you used and the **full** compile output here
  ``` I just double clicked the XMR-Stak .exe
  ```
  run `cmake -LA .` in the build folder and add the output here
  ``` I don't think "cmake" command works for Windows

# Issue with the execution
  - Do you compiled the miner by our own?
  ```
  run `./xmr-stak --version-long` and add the output here
  ```

# AMD OpenCl issue

  ```
  run `clinfo` and add the output here
  ``` I did try run 'clinfo" in Command Prompt and it crashed too.

# Stability issue
  - Is the CPU or GPU overclocked?
     No
  - Is the Main memory of the CPU or GPU undervolted?
     Don't know. Currently I have one 1000W Corsair PSU in use.

**Comment by psychocrypt** [comment]
please post all miner output!

leokillbill <notifications@github.com> schrieb am Fr., 30. März 2018, 13:28:

**Comment by leokillbill** [comment]
Sorry @psychocrypt, my miner doesn't run at all and just crash. But I figured it out now. 
After I installed my first Vega, the Windows Device Manager only shows that, and I think the System has automatically muted the internal graphic card. 
I went back to BIOS and set the system default to the internal Intel Graphic card. Now the Device Manager shows both Intel and Vega cards. And the Windows is using the internal Intel card to display. 
For some reason, this fixed the miner crash issue and now it started started to discover the blocks.
Thanks @psychocrypt for the awesome miner! I was using 2.2 and will try the latest 2.3 soon.
Thanks.
